### PR TITLE
Admin show edit users role

### PR DESCRIPTION
**Pull request by huyhoc1310**
https://edu-redmine.framgia.vn/issues/559

### CODE REVIEW

**Comment by huyhoc1310** [comment]
ready

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Admin::UsersController < ApplicationController
+  before_action :is_admin?
+  before_action :find_user, only: %i(edit update destroy)
+
+  def index
+    @users = User.load_data.page(params[:page]).per Settings.rows
+  end
+
+  def update
+    @user.update_columns role: params[:role]
```
Comment (huongnguyenmta) [inline app/controllers/admin/users_controller.rb]: check update fail

**Comment by huongnguyenmta** [comment]
commented

**Comment by huyhoc1310** [comment]
ready

**Pull request merged**